core/scripts/i18n/prune-obsolete-lhl-messages.js

```diff
@@ -69,6 +69,10 @@ function pruneLocale(goldenLocaleArgumentIds, localeLhl, alreadyLoggedPrunes) {
   const remainingMessages = {};
 
   for (const [messageId, {message}] of Object.entries(localeLhl)) {
+    if (messageId.startsWith('node_modules/@paulirish/trace_engine')) {
+      continue;
+    }
```

Comment:
Is there any harm in pruning them? If there's a placeholder mismatch between the UIStrings version and a particular locale, formatting for that locale will end up throwing an error. Pruning prevents that, even if it's unlikely a mismatch would end up in trace-engine (pruning and injecting would also have to swap in collect-strings)